lib/ex_aws/config.ex

```diff
@@ -42,6 +42,17 @@ defmodule ExAws.Config do
     |> parse_host_for_region
   end
 
+  @doc """
+  Builds a minimal HTTP configuration.
+  """
+  def http_config(service, opts \\ []) do
```

Comment:
I like this approach, opens the door for anything else that might need it in the future.